### DISCUSSION TITLE
Can a developer switch to a business "role" without having to create a business account?

### DISCUSSION THREAD

**Discussion by Rex-am-I** [joemasilotti/railsdevs.com#425] [Q&A] [answered]
Given that a significant number of developers turn into entrepreneurs really quick, it is not inconceivable that some may start seeking fellow dev's either as employees or co-founders. Is it possible to do that right now without said dev having to create a business account?

**Comment by joemasilotti** [comment, marked as answer]
Yep! It's hidden from the UI if you already have a developer profile. You can create your business one here: https://railsdevs.com/role/new 

Happy to accept a PR if there's a sane place to add it to the UI.
